src/StatusQ/Popups/StatusSearchPopup.qml

```diff
@@ -0,0 +1,198 @@
+    property string selectionBadgeImage: ""
+    property string selectionBadgePrimaryText: ""
+    property string selectionBadgeSecondaryText: ""
+    property string selectionBadgeIdenticonColor: ""
+    property bool loading
+    property Component searchOptionsPopupMenu
+    property ListModel searchResults: ListModel { }
+    function resetSelectionBadge() {
+        selectionBadgeImage = "";
+        selectionBadgePrimaryText = "Anywhere";
+        selectionBadgeSecondaryText = "";
+        selectionBadgeIdenticonColor = "";
+    }
+
+    content: Item {
+        width: parent.width
+        height: root.height
+        property alias searchText: inputText.input.text
+
+        ColumnLayout {
+            id: contentItemColumn
+            anchors.fill: parent
+            spacing: 0
+            Item {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 63
+                StatusInput {
+                    id: inputText
+                    anchors.left: parent.left
+                    anchors.leftMargin: 8
+                    anchors.right: parent.right
+                    anchors.rightMargin: 8
+                    anchors.bottom: parent.bottom
+                    input.icon.name: "search"
+                    input.placeholderText: "Search"
+                    input.bgVisible: false
+                }
+            }
+            StatusMenuSeparator { Layout.fillWidth: true }
+            Item {
+                Layout.preferredWidth: parent.width
+                Layout.preferredHeight: 58
+                StatusButton {
+                    id: searchOptionsMenuButton
+                    anchors.left: parent.left
+                    anchors.leftMargin: 16
+                    anchors.verticalCenter: parent.verticalCenter
+                    implicitWidth: (contentItemRowLayout.width + 8)
+                    implicitHeight: 32
+
+                    contentItem: Item {
+                        anchors.fill: parent
+                        RowLayout {
+                            id: contentItemRowLayout
+                            anchors.centerIn: parent
+                            spacing: 2
+                            StatusBaseText {
+                                text: "In: "
+                            }
+                            StatusRoundedImage {
+                                implicitWidth: 16
+                                implicitHeight: 16
+                                visible: !!root.selectionBadgeImage
+                                image.source: root.selectionBadgeImage
+                            }
+                            StatusLetterIdenticon {
+                                implicitWidth: 16
+                                implicitHeight: 16
+                                letterSize: 11
+                                visible: ((root.selectionBadgeImage === "") && (root.selectionBadgePrimaryText !== "Anywhere"))
```

Comment:
Can't rely on `"Anywhere"` here as the text could be in a different language when i18n'ed.

Do we have other heuristics?

Reply:
done